### ISSUE TITLE
multi label classifier

### ISSUE THREAD

**Issue by francescoagati** [cazala/synaptic#215]
hi,
there is an example of multi label classifier in synaptic?

**Comment by Jabher** [comment]
No, it is not. And also you will need to implement your own cost function, Binary Crossentropy, which is not very compicated, but you need to keep it in mind that there is no bundled functions for this functionality.

**Comment by cazala** [comment]
I don't know if this helps, but the DSR example is a classification problem that uses cross entropy as cost function: http://caza.la/synaptic/#/dsr

**Comment by wagenaartje** [comment]
@Jabher @cazala why is another cost function necessary? I went through synaptic code and the cost function in no way effects the backpropagation algorithm. In synaptic, the cost function is only used as a measurement for the trainer to see how well trainer a network is - but a network does not try to optimize the given cost function, have a look:

https://jsfiddle.net/wagenaartje/dguhLo6y/4/

Open console. (there is double code but just to proof that the cost function has no effect on the backpropagation).

**Comment by cazala** [comment]
It is only used to compute the error during training
